tests/e2e/scripts/cilium_nokubeproxy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+ip4_addr=$1
+
+# Override default calico and specify the interface for windows-agent
+# by default, the windows-agent use a different interface name than the linux-agent
+mkdir -p /var/lib/rancher/rke2/server/manifests
```

Comment:
You mention the `windows-agent`, but this test doesn't have any windows VMs. Is the intention to expand this test later?

Reply:
copy/paste typo